spinalcordtoolbox/reports/qc.py

```diff
@@ -62,22 +63,27 @@ class QcImage(object):
                      "#a22abd", "#d58240", "#ac2aff"]
     # _seg_colormap = plt.cm.autumn
 
-    def __init__(self, qc_report, interpolation, action_list, stretch_contrast=True,
-                 stretch_contrast_method='contrast_stretching', angle_line=None):
```

Comment:
Ah, one other comment. Could you double check that adding the `process` parameter like this meshes with the existing usage of `QcImage`? For example:

https://github.com/neuropoly/spinalcordtoolbox/blob/916ab591ca2e35cab9f5f27ec2439ce8fdb0b9d2/spinalcordtoolbox/reports/qc.py#L565-L566